src/components/ebay-breadcrumb/test/test.server.js

```diff
@@ -0,0 +1,65 @@
+const expect = require('chai').expect;
+const testUtils = require('../../../common/test-utils/server');
+const mock = require('../mock/index');
+
+test('shoudl return zero nav element in the page', context => {
+    const input = {};
+    const $ = testUtils.getCheerio(context.render(input));
+    expect($('nav').length).to.equals(0);
+});
+test('should return nav element with input', context => {
+    const $ = testUtils.getCheerio(context.render(mock.items));
+    const nav = $('nav');
+    expect(nav.length).to.equals(1);
+    expect(nav.attr('role')).to.equals('navigation');
+    expect(nav.attr('dataview')).to.equals('data-view tracking');
+    expect(nav.attr('class')).to.equals('breadcrumb');
+    expect(nav.attr('aria-labelledby')).to.equals('breadcrumbs_heading');
+});
+test('should return h2 tag', context => {
+    const $ = testUtils.getCheerio(context.render(mock.items));
+    const h2Tag = $('h2');
+    expect(h2Tag.length).to.equals(1);
+    expect(h2Tag.attr('id')).to.equals('breadcrumbs_heading');
+    expect(h2Tag.attr('class')).to.equals('clipped');
+    expect(h2Tag.html()).to.equals('You are here');
+});
+test('should return 4 li with anchor', context => {
```

Comment:
These first four tests should be combined, or refactored to be more DRY. There's a ton of duplication of the rendering of `mock.items`.